controller/controlplane/controller_reconciler_utils.go

```diff
@@ -143,6 +143,11 @@ type ensureDeploymentParams struct {
 	ServiceAccountName             string
 	AdminMTLSCertSecretName        string
 	AdmissionWebhookCertSecretName string
+	// EnforceConfig is a flag to enforce the configuration of the Deployment.
+	// If set to true, the Deployment will be updated even if the spec hash matches.
+	// This is useful when the Deployment has been manually modified by something
+	// other than the operator to prevent the operator from endless reconciliation.
```

Comment:
```suggestion
	// other than the operator (typically mutation webhook that enforces some cluster-wide policy, typically for resources or security) to prevent the operator from endless reconciliation.
```
to give even more hint to the future developers

Reply:
```suggestion
	// other than the operator to prevent the operator from endless reconciliation
	// (typically mutation webhook that enforces some cluster-wide policy, 
	// typically for resources or security).
```